packages/framework/ini-files/config-specs.ini

```diff
@@ -1872,50 +1858,27 @@ opt-set-cmake-var MueLu_Structured_Interp_SA_Laplace2D_kokkos_MPI_4_DISABLE BOOL
 use rhel8_oneapi-intelmpi_release-debug_shared_no-kokkos-arch_no-asan_no-complex_fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
 use PACKAGE-ENABLES|ALL
 
-[ubuntu_gnu_release-debug_shared_no-kokkos-arch_no-asan_no-complex_fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_all]
-use BUILD-TYPE|RELEASE-DEBUG
-use KOKKOS-ARCH|NO-KOKKOS-ARCH
-use USE-ASAN|NO
-use USE-COMPLEX|NO
-use USE-FPIC|YES
-
-opt-set-cmake-var BUILD_SHARED_LIBS      BOOL   : ON
-
-opt-set-cmake-var Trilinos_ENABLE_TESTS BOOL : ON
-
-opt-set-cmake-var CMAKE_C_COMPILER       FILEPATH : ${CC|ENV}
-opt-set-cmake-var CMAKE_CXX_COMPILER     FILEPATH : ${CXX|ENV}
-opt-set-cmake-var CMAKE_Fortran_COMPILER FILEPATH : ${FC|ENV}
-opt-set-cmake-var TPL_ENABLE_MPI         BOOL     : OFF
-
-use USE-PT|NO
-use USE-RDC|NO
-use USE-UVM|NO
-use USE-DEPRECATED|YES
-
-opt-set-cmake-var Trilinos_ENABLE_Teuchos BOOL : ON
-opt-set-cmake-var Trilinos_ENABLE_ALL_PACKAGES BOOL FORCE : OFF
-
 [rhel8_cuda-gcc-openmpi_release_static_Ampere80_no-asan_complex_no-fpic_mpi_pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
+use COMMON_SPACK_TPLS
+# Not using currently to prevent warning overload
+#use COMPILER|GNU
 use NODE-TYPE|CUDA
 use BUILD-TYPE|RELEASE
-opt-set-cmake-var BUILD_SHARED_LIBS      BOOL   : OFF
-opt-set-cmake-var TPL_FIND_SHARED_LIBS   BOOL   : OFF
+use LIB-TYPE|STATIC
 use KOKKOS-ARCH|AMPERE80
 use USE-ASAN|NO
+use USE-COMPLEX|YES
 use USE-FPIC|NO
 use USE-MPI-NO-EXPLICIT-COMPILERS|YES
 use USE-PT|NO
-use USE-COMPLEX|YES
 use USE-RDC|NO
 use USE-UVM|NO
 use USE-DEPRECATED|YES
 use PACKAGE-ENABLES|NO-EPETRA
+
 use CUDA
 use SPACK_NETLIB_BLAS_LAPACK
 
-opt-set-cmake-var Trilinos_ENABLE_TESTS BOOL : ON
```

Comment:
Was this supposed to get enabled else where for non-uvm?

Reply:
Yes, it's up further in the file (comes in via `COMMON`)